guide/keyv/README.md

```diff
@@ -101,7 +101,7 @@ client.on('message', async message => {
 			if (message.content.startsWith(guildPrefix)) prefix = guildPrefix;
 		}
 
-		// if we found a prefix, setup args; otherwise, this isn't a command
+		// if we found a prefix, setup args. Otherwise, this isn't a command
```

Comment:
Isn't this already correct?
```suggestion
		// if we found a prefix, setup args; otherwise, this isn't a command
```

Reply:
How would it be? Isn't that a bit incorrect grammar

Reply:
initial was correct